lib/expand/join.go

```diff
@@ -512,45 +538,44 @@ func (p *Peer) printStep(format string, args ...interface{}) {
 	p.dispatcher.Send(event)
 }
 
-// createExpandOperation creates a new expand operation
-func (p *Peer) createExpandOperation(operator ops.Operator, cluster ops.Site) (*ops.SiteOperation, error) {
-	key, err := operator.CreateSiteExpandOperation(p.ctx, ops.CreateSiteExpandOperationRequest{
-		AccountID:   cluster.AccountID,
-		SiteDomain:  cluster.Domain,
-		Provisioner: schema.ProvisionerOnPrem,
-		Servers:     map[string]int{p.Role: 1},
-	})
+// dialWizard connects to a wizard
+func (p *Peer) dialWizard(addr string) (*operationContext, error) {
```

Comment:
This methods seems to just be wrapping `connectWizard` as-is, can we get rid of it and just use `connectWizard` (or rename `connectWizard` to `dialWizard` and use the "dial" one).

Reply:
Yup - it used to be more but degenerated to `connectWizard` which I'll merge back as `dialWizard`.